### PR TITLE
Fix: LCFS Bug ag-grid copy paste function

### PR DESCRIPTION
**Pull request by prv-proton**
Fixes: #1301, #1302

### CODE REVIEW

File: frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx [hunk 1]
```diff
@@ -78,12 +78,19 @@ export const fuelCodeColDefs = (optionsData, errors, isCreate, canEdit) => [
     valueGetter: (params) => {
       if (params.data?.fuelCodePrefix?.prefix) {
         return params.data.fuelCodePrefix.prefix
-      } else if (params.data.prefixId) {
+      }else if (params.data.prefixId) {
```
Comment (dhaselhan) [inline frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx:81]: spacing

File: frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx [hunk 2]
```diff
@@ -78,12 +78,19 @@ export const fuelCodeColDefs = (optionsData, errors, isCreate, canEdit) => [
...
         const selectedOption = optionsData?.fuelCodePrefixes?.find(
           (obj) => obj.fuelCodePrefixId === params.data.prefixId
         )
         return selectedOption.prefix
       }
+      const selectedOption = optionsData?.fuelCodePrefixes?.find(
+        (obj) => obj.prefix === params.data.prefix
+      )
+      if (params.data.prefix) {
```
Comment (dhaselhan) [inline frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx:90]: Looks like selectedOption could be undefined here due to the optional chaining. Should this be if (selectedOption) ?

**Review by dhaselhan — CHANGES_REQUESTED** [review]

**Comment by prv-proton** [comment]
Thanks @dhaselhan, fixed your comments.

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**